lib/rspec_api_documentation/writers/formatter.rb

```diff
@@ -3,9 +3,9 @@ module Writers
     module Formatter
 
       def self.to_json(object)
-        JSON.pretty_generate(object.as_json)
+        JSON.generate(object.as_json)
```

Comment:
Of course it isn't totally desirable. Hence why this was never merged.

But I can confirm that I'm no longer seeing JRuby related issues as it seems that the underlying issues have been fixed.

@jakehow Maybe it's time to close this PR without merging it in?